client/src/components/VideoCards.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Buttons from "./Buttons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrashAlt } from "@fortawesome/free-solid-svg-icons";
+
+const VideoCards = ({ videoData, setVideoData }) => {
```

Comment:
nice that you've separated these into a component, you could go further & have a component for each video card